.vale/styles/Localization/Modals.yml

```diff
@@ -0,0 +1,15 @@
+---
+extends: existence
+ignorecase: true
+level: warning
+link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/termswarnings/
+message: "Avoid terms that are ambiguous and problematic in a localization context"
```

Comment:
Maybe make more specific to modals: "Avoid using modals because they can introduce uncertainty"

Reply:
Updated as per suggestion! Thanks!